crates/red_knot/src/symbols.rs

```diff
@@ -721,27 +771,27 @@ mod tests {
     fn insert_same_name_symbol_twice() {
         let mut table = SymbolTable::new();
         let root_scope_id = SymbolTable::root_scope_id();
-        let symbol_id_1 = table.add_symbol_to_scope(root_scope_id, "foo");
-        let symbol_id_2 = table.add_symbol_to_scope(root_scope_id, "foo");
+        let symbol_id_1 = table.add_symbol_to_scope(root_scope_id, "foo", SymbolFlags::empty());
```

Comment:
Would be good to also add/modify tests above here to actually verify flags are set correctly

Reply:
I added a few assertions to existing tests. If that's not alright, lmk and I'll make distinct tests.